src/daemon/utils_cache.c

```diff
@@ -526,46 +553,51 @@ int uc_get_value(metric_t const *m, value_t *ret) {
   return status;
 } /* value_t *uc_get_value */
 
-static int uc_get_first_time_by_name(const char *name, cdtime_t *ret_time) {
+static uc_first_metric_result_t uc_first_metric_by_name(const char *name) {
```

Comment:
As above: I don't think the verb adds anything to the function name.

My medium-term plan is to change the key to use a struct instead of a string, which will make this function obsolete.